src/framework/control.ml

```diff
@@ -359,6 +374,7 @@ struct
     in
 
     let entrystates = List.map (fun (n,e) -> (MyCFG.FunctionEntry n, Spec.context e), e) startvars in
+    let entrystates_global = GHT.to_list gh in
```

Comment:
Could you explain why this works with `exp.earlyglobs` on, whereas it did not when `GHT.to_list gh` to list was called in `do_global_inits`?
This strikes me as odd

Reply:
I think it might still work if left there, but since I moved `gh` out of `do_global_inits` for `enter_with`, then it also made sense to move the final transformation of `gh` out of there. Now both `entrystates` are defined in the same location.